internal/oci/annotations.go

```diff
@@ -36,10 +36,15 @@ func ProcessAnnotations(ctx context.Context, s *specs.Spec) (err error) {
 	// expand annotations
 	for key, exps := range annotations.AnnotationExpansions {
 		// check if annotation is present
-		if val, ok := s.Annotations[key]; ok {
+		if expVal, ok := s.Annotations[key]; ok {
+			// If the annotation is a bool and is set to false, we will treat that as
+			// equivalent to disabling the expansion.
+			if strings.ToLower(expVal) == "false" {
+				continue
+			}
```

Comment:
this doesn't match the doc comment for `AnnotationExpansions`: 
```
It is an error for the expansions to already exist and have a value that differs from the value set here.
```
Also, we are assuming here that `AnnotationExpansions` has truthy semantics, when the comment and logic just assumes that the expansions have the same value as the original annotation

Reply:
I think I had changed that comment and what I meant by "a value that differs from the value set here" is that the child annotation's value set in the expansion definition cannot conflict with whatever is put on the spec. 

What do you mean by "truthy" semantics?